dart/lib/src/sentry_options.dart

```diff
@@ -136,7 +136,7 @@ class SentryOptions {
   // TODO: those ctor params could be set on Sentry._setDefaultConfiguration or instantiate by default here
   SentryOptions({
     this.dsn,
-    this.compressPayload,
+    this.compressPayload = true,
     this.httpClient,
     ClockProvider clock = getUtcDateTime,
   }) {
```

Comment:
please remove this ctor and add setters if not there yet.
`clock` has default value `getUtcDateTime`
`compressPayload` is `true`
`compressPayload` probably a NoOp if its platform dependent

Reply:
we can keep the `dsn` as optional/named param on the ctor though, seems useful

Reply:
✅